src/server/components/auth/authenticated-user.ts

```diff
@@ -50,6 +43,38 @@ export default class AuthenticatedUser {
     return lists ?? [];
   }
 
+  static calculateSortOrder(customSortOrder?: Record<string, string>): Array<Record<string, string>> {
```

Comment:
`authenticatedUser.getLists` suggests "get the lists associated with the authenticatedUser" but `calculateSortOrder` should really be done/handled by the request, not the User class. (User should not know how to parse query parameters)